third_party/mediapipe_calculators/BUILD

```diff
@@ -0,0 +1,121 @@
+        "@google_mediapipe//mediapipe/calculators/core:concatenate_detection_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:concatenate_proto_list_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:clip_vector_size_calculator",
+        #SEGFAULT"@google_mediapipe//mediapipe/calculators/core:clip_detection_vector_size_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:counting_source_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:make_pair_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:matrix_multiply_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:matrix_subtract_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:mux_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:non_zero_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_cloner_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_inner_join_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_thinner_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:pass_through_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:round_robin_demux_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:immediate_mux_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_presence_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:string_to_int_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:default_side_packet_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_resampler_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:split_proto_list_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:dequantize_byte_array_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:quantize_float_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:sequence_shift_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:gate_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:matrix_to_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:merge_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:stream_to_side_packet_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:constant_side_packet_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:graph_profile_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:get_vector_item_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:vector_size_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:packet_sequencer_calculator",
+        "@google_mediapipe//mediapipe/calculators/core:merge_to_vector_calculator",
+        "@google_mediapipe//mediapipe/calculators/tflite:ssd_anchors_calculator",
```

Comment:
Why do we need tflite calculator? Is it used in POC?

Reply:
Yes